frontend/e2e/navigate-to-temporal-data-page-and-test-content.spec.ts

```diff
@@ -1,97 +1,89 @@
+      await expect(page).toHaveURL(
+        `/amending-laws/eli/bund/bgbl-1/2017/s419/2017-03-15/1/deu/regelungstext-1/temporal-data`,
+      )
+
+      await expect(
+        page.getByRole("heading", { level: 1, name: "Zeitgrenzen anlegen" }),
+      ).toBeVisible()
+    })
   })
-})
 
-test.describe("management of Temporal Data for an amending law", () => {
-  async function setupInitialData(page: Page) {
-    await page.request.put(
-      "/api/v1/norms/eli/bund/bgbl-1/2017/s419/2017-03-15/1/deu/regelungstext-1/timeBoundaries",
-      {
-        headers: {
-          "Content-Type": "application/json",
-          Accept: "application/json",
-        },
-        data: JSON.stringify([
-          { date: "2017-03-16", eventRefEid: "meta-1_lebzykl-1_ereignis-2" },
-        ]),
-      },
-    )
-  }
-
-  test(`adding, editing, deleting and saving time boundaries`, async ({
-    page,
-  }) => {
-    await page.goto(
-      "/amending-laws/eli/bund/bgbl-1/2017/s419/2017-03-15/1/deu/regelungstext-1/temporal-data",
-    )
-    await setupInitialData(page)
-
-    // check contents of entry into force article html rendering
-    await expect(page.getByText("Artikel 3Inkrafttreten")).toBeVisible()
-
-    const saveButton = page.locator("text=Speichern")
-    let dateInputs = page.locator('[data-testid="date-input-field"]')
-
-    await expect(dateInputs).toHaveCount(1)
-    await expect(dateInputs).toHaveValue("16.03.2017")
-
-    //   add new time boundaries
-    const newDateInput = page.locator('[data-testid="new-date-input-field"]')
-    await newDateInput.fill("01.05.2023")
-    await newDateInput.fill("02.06.2023")
-    await saveButton.click()
-
-    await page.reload()
-    await expect(dateInputs).toHaveCount(3)
-
-    // validate each date input contains the correct data
-    await expect(dateInputs.nth(0)).toHaveValue("16.03.2017")
-    await expect(dateInputs.nth(1)).toHaveValue("01.05.2023")
-    await expect(dateInputs.nth(2)).toHaveValue("02.06.2023")
-
-    //edit time boundaries
-    const dateInputToEdit = page
-      .locator('[data-testid="date-input-field"]')
-      .nth(1)
-    await dateInputToEdit.fill("03.06.2023")
-    await saveButton.click()
-    await page.reload()
-
-    await expect(dateInputs.nth(1)).toHaveValue("03.06.2023")
-
-    //   delete time boundaries
-    for (let i = 2; i > 0; i--) {
-      const deleteButton = page.locator(`[data-testid="delete-button-${i}"]`)
-      await deleteButton.click()
+  test.describe("management of Temporal Data for an amending law", () => {
+    test(`adding, editing, deleting and saving time boundaries`, async ({
+      page,
+    }) => {
+      await page.goto(
+        "/amending-laws/eli/bund/bgbl-1/2017/s419/2017-03-15/1/deu/regelungstext-1/temporal-data",
+      )
+      // check contents of entry into force article html rendering
+      await expect(page.getByText("Artikel 3Inkrafttreten")).toBeVisible()
+
+      const saveButton = page.locator("text=Speichern")
```

Comment:
Note: This is not related to the SQL changes but some general feedback on the selectors: We should avoid using `page.locator` when possible. The things it's doing here can just as well be accomplished by a more readable and precise selector (e.g. `getByText` and `getByTestId`, or in this case even better: `getByRole("button", { name: "Speichern" })`). This is the [recommended best practice](https://playwright.dev/docs/locators#quick-guide) by Playwright whereas using `locator` is discouraged unless there is no other way to do it. Also see this [thread](https://digitalservicebund.slack.com/archives/C046VD44ZEH/p1715750128424729) in Slack for context.